.github/scripts/windows/build.bat

```diff
@@ -5,7 +5,7 @@ if /i "%GITHUB_ACTIONS%" neq "True" (
     exit /b 3
 )
 
-set SDK_REMOTE=https://github.com/php/php-sdk-binary-tools.git
+set SDK_REMOTE=https://github.com/bukka/php-sdk-binary-tools.git
```

Comment:
I'm concerned about depending on a repository outside of the PHP organization. Is it not possible to update the original repo?

Reply:
This is just a test.

Reply:
I will change it to draft